perun-wui-registrar/src/main/resources/cz/metacentrum/perun/wui/registrar/client/resources/PerunRegistrarTranslation_cs.properties

```diff
@@ -1,177 +1,181 @@
 # // -------------- APP NAME------------------------ //
 
 registrarAppName=Registrace
-pleaseVerifyCaptcha=Pro pokračování prosím proveďte potvrzení níže
-captchaFailed=Ověření selhalo. Prosím, proveďte potvrzení znovu.
+pleaseVerifyCaptcha=Pro pokra\u00C4\u008Dov\u00C3\u00A1n\u00C3\u00AD pros\u00C3\u00ADm prove\u00C4\u008Fte potvrzen\u00C3\u00AD n\u00C3\u00AD\u00C5\u00BEe
```

Comment:
Our source files should be in UTF-8 so there is no need to convert the messages. I think you should update your IDE settings to respect UTF-8 for this file (IDE usually does this transparent translation and think these files are ASCII only).

Reply:
Thanks, did not notice. For some reason when I tried to switch to UTF-8, it re-encoded the whole file, so I left it as it was. Seems it did the re-encoding to ISOwhatever on the fly without marking changes.